functions/New-DbaSsisCatalog.ps1

```diff
@@ -0,0 +1,117 @@
+After installing the SQL Server Engine and SSIS you still have to enable the SSIS Catalog. This function will enable the catalog and gives the option of supplying the password. 
+
+.PARAMETER SqlServer
+SQL Server you wish to run the function on.
+
+.PARAMETER SqlCredential
+Credenitals used to connect to the SQL Server
+
+.PARAMETER SsisCredential
+Optional Password that will be used for the security key in SSISDB
+
+.NOTES 
+Author: Chrissy LeMaire (@cl), netnerds.net
+Requires: sysadmin access on SQL Servers
+
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/New-DbaSsisCatalog
+
+.EXAMPLE   
+New-DbaSsisCatalog -SqlServer DEV01 -SsisCredential MySuperPassword
+
+Creates the SSIS Catalog on server DEV01
+
+#>
+	[CmdletBinding(SupportsShouldProcess = $true)]
+	Param (
+		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlInstance")]
+		[string[]]$SqlServer,
+        [System.Management.Automation.PSCredential]$SqlCredential,
+        [string]$SsisPassword
+
+        )
+	PROCESS
+	{
+        foreach ($Instance in $SqlServer)
+		{
+            Write-Verbose "Attempting to connect to $Instance"
+			try
+			{
+				$server = Connect-SqlServer -SqlServer $Instance -SqlCredential $SqlCredential
+			}
+			catch
+			{
+				Write-Warning "Can't connect to $Instance or access denied. Skipping."
+				continue
+			}
+
+            ## $SsisCredential.GetNetworkCredential().password
+
+            #if SQL 2012 or higher only validate databases with ContainmentType = NONE
+            if (!($server.versionMajor -gt 10))
+		    {
+                write-Warning "This version of SQL Server cannot have SSIS catalog"
+                break
+            }
+            ## need to check if ssis installed
+            if ($true)
+            {
+                Write-Warning "You do not have SSIS installed on this instance"
+            }
+
+            ## check if SSIS and Engine running on box
+            $services = Get-DbaSqlService -ComputerName $server.NetName 
+            $ssis_scv = $Services | Where {$_.ServiceType -eq "SSIS" -and $_.State -eq "Running"} 
+            
+            if (-not $ssis_scv)
+            {
+                write-warning "SSIS is not running on the server you supplied"
+                break
```

Comment:
This should be continue so it does not cause the function to stop...which think we have some new internal functions now instead of using break and such.